src/mindlessgen/cli/cli_parser.py

```diff
@@ -195,19 +201,19 @@ def cli_parser(argv: Sequence[str] | None = None) -> dict:
         required=False,
         help="Path to the xTB binary.",
     )
+    parser.add_argument(
```

Comment:
Why did you shift the different entries in `cli_parser`?

Reply:
Because there was a ToDo that said "add the GFN level here".

Reply:
That was just a remnant that I had forgotten to delete. 😂 But it's okay, so just leave as it is currently the case here (doesn't matter anyway).